hera_sim/components.py

```diff
@@ -37,7 +38,7 @@ class determine how to apply the effect simulated by
     #: Whether this systematic multiplies existing visibilities
     is_multiplicative: bool = False
 
-    _alias: Tuple[str] = tuple()
+    _alias: tuple[str] = tuple()
```

Comment:
Does the change from `Tuple` to `tuple` in the typing have any effect? It looks like one of the new style-enforcing features you added changed `Tuple` to `tuple` and `Dict` to `dict` wherever those occur.

Reply:
No, no effect, just one less import to make. You need I think py38 to use it.